charts/ocis/templates/antivirus/deployment.yaml

```diff
@@ -75,6 +80,10 @@ spec:
             - name: messaging-system-ca
               mountPath: /etc/ocis/messaging-system-ca
               readOnly: true
+            {{- if .Values.customCAChain.enabled }}
+            - name: custom-ca-chain
+              mountPath: /etc/ssl/certs/
+            {{- end }}
```

Comment:
Would be good to have this fragment templated so we don't have these lines in all `deployments.yml` file but only one with a call here.

suggestion

```
{{/*
oCIS ca mount

@param .path          The path to mount
*/}}
{{- define "ocis.caPath" -}}
            {{- if .Values.customCAChain.enabled }}
            - name: custom-ca-chain
              mountPath: {{ .path }}
            {{- end }}
{{- end -}}
```

Usage afterwards with `{{- include "ocis.caPath" /etc/ssl/certs }}`